pkg/jobcreator/controller.go

```diff
@@ -145,8 +149,29 @@ func (controller *JobCreatorController) subscribeToWeb3() error {
 }
 
 func (controller *JobCreatorController) Start(ctx context.Context, cm *system.CleanupManager) chan error {
+	// Initial fetch of the module allowlist
+	err := controller.UpdateModuleAllowlist()
+	if err != nil {
+		// handle error
```

Comment:
How do we plan to handle this error?

Reply:
I passed this to the controller with contoller.log.Error... Does this work? Especially since this error is mainly for testing, passing it to web3 metrics is overkill?

Reply:
Yeah, agreed passing it to metrics probably isn't needed. We'd probably get a bunch of reports whenever someone typos cowsay. 😆

Let's chat about the `controller.log.Error` approach. It's probably right, but would love to chat about it because I am still getting familiar with the code base.